core/libp2p/transport/tcp/tcp_connection.hpp

```diff
@@ -43,6 +43,8 @@ namespace libp2p::transport {
     void asyncRead(boost::asio::mutable_buffer &mut, uint32_t to_read,
                    std::function<Readable::CompletionHandler> cb) override;
 
+    std::async
```

Comment:
?

Reply:
I assume this is left after showing some example in the code. Going to be removed